api/controllers/deploy.go

```diff
@@ -51,12 +54,12 @@ func (d *DeployController) CreateDeploy(c *fireball.Context) (fireball.Response,
 		return nil, errors.New(errors.InvalidRequest, err)
 	}
 
-	return createJob(d.JobStore, job.CreateDeployJob, req)
+	return createJob(d.TagStore, d.JobStore, job.CreateDeployJob, req)
```

Comment:
was this a bug in the prior implementation to not include `TagStore`?

Reply:
yes, we weren't creating tags when we would create a job, so we weren't able to resolve job tags.

Reply:
ah right, now I remember this being mentioned previously. 🏅